api/version.swagger.yaml

```diff
@@ -0,0 +1,274 @@
+swagger: "2.0"
```

Comment:
I think we generate this file in Makefile

Reply:
yes, and then we copy it there to be able to distribute it to clients